Dockerfile

```diff
@@ -40,4 +42,4 @@ COPY ./ $MAGPIE_DIR
 # equivalent of `make install` without conda env and pre-installed packages
 RUN pip install --no-dependencies -e $MAGPIE_DIR
 # equivalent of `make cron start` without conda env
-CMD crond && gunicorn -b 0.0.0.0:2001 --paste $MAGPIE_DIR/config/magpie.ini --workers 10 --preload
+CMD crond -c $CRON_DIR && gunicorn -b 0.0.0.0:2001 --paste $MAGPIE_DIR/config/magpie.ini --workers 10 --preload
```

Comment:
Should use `MAGPIE_CONFIG_DIR` in case it is overridden.

Reply:
Look at the `CRON_DIR` variable in the Dockerfile, I think `MAGPIE_CONFIG_DIR` refers to something else.

Reply:
I meant in the `--paste` parameter.